process/elasticproc/converters/balanceToFloat.go

```diff
@@ -74,7 +78,12 @@ func (bc *balanceConverter) computeBalanceAsFloat(balance *big.Int, balancePreci
 
 	balanceFloatWithDecimals := math.Round(bal*balancePrecision) / balancePrecision
 
-	return core.MaxFloat64(balanceFloatWithDecimals, 0)
+	value := core.MaxFloat64(balanceFloatWithDecimals, 0)
+	if math.IsInf(value, +1) || math.IsInf(value, -1) {
+		return 0
```

Comment:
is it ok to return 0 here? The reason for the question is that, in case of very large numbers, you will return 0 here and you will have a balances mismatch. The correct approach here was to return an error.

Reply:
refactored this component.